engine/flink/api/src/main/scala/pl/touk/nussknacker/engine/flink/api/process/FlinkLazyParameterFunctionHelper.scala

```diff
@@ -4,12 +4,13 @@ import org.apache.flink.api.common.functions.{RuntimeContext, _}
 import org.apache.flink.configuration.Configuration
 import org.apache.flink.util.Collector
 import pl.touk.nussknacker.engine.api._
+import pl.touk.nussknacker.engine.api.exception.ExceptionComponentInfo
 import pl.touk.nussknacker.engine.flink.api.exception.ExceptionHandler
 
 /*
   This is helper class that allows to evaluate LazyParameter[T] in Flink functions.
  */
-class FlinkLazyParameterFunctionHelper(val nodeId: String,
+class FlinkLazyParameterFunctionHelper(val exceptionComponentInfo: ExceptionComponentInfo,
```

Comment:
This class should be named other way. `nodeId` wasn't used only for porpose of exception handling. In the `FlinkLazyParameterFunctionHelper` we have utilities for component implementations. Maybe the name should rather shows "what this class is" instead of "how it is usage". IMO this class is basically bunch of crusial identifiers related to component usage (which we call "node"). WTYT about `NodeIdentifiers` or `NodeComponentId`. Any other ideas?

Reply:
`NodeIdentifiers` suggests that there are ids of many different nodes. `NodeComponentId` is good :)